Tests/Runtime/MeshOps/AppendElementsTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using UnityEngine;
+using UObject = UnityEngine.Object;
+using NUnit.Framework;
+using UnityEngine.ProBuilder.Tests.Framework;
+using UnityEngine.ProBuilder.MeshOperations;
+
+namespace UnityEngine.ProBuilder.RuntimeTests.MeshOperations
+{
+    class AppendElementsTests
+    {
+        GameObject m_GO;
+        PolyShape m_Poly;
+        ProBuilderMesh m_pb;
+
+        [SetUp]
+        public void SetUp()
+        {
+            m_GO = new GameObject();
+            m_Poly = m_GO.AddComponent<PolyShape>();
+            m_pb = m_GO.AddComponent<ProBuilderMesh>();
+        }
+
+        [TearDown]
+        public void Cleanup()
+        {
+            UObject.DestroyImmediate(m_GO);
+        }
+
+        [Test]
+        public void CreateShapeFromPolygon_ExtrudeOrNot()
+        {
+            m_Poly.m_Points.Add(new Vector3(0,0,0));
+            m_Poly.m_Points.Add(new Vector3(0,0,2));
+            m_Poly.m_Points.Add(new Vector3(2,0,0));
+            m_Poly.extrude = 0.0f;
+            m_Poly.flipNormals = false;
```

Comment:
what happens if flipNormals is true? is it possible for it to be?

Reply:
It is just used when creating the opposite face. actually if I do not initialize it is the default value false .